### ISSUE TITLE
Question: Dose this Support Blur

### ISSUE THREAD

**Issue by AurevoirXavier** [koekeishiya/yabai#493]
If not, any plan to implement it?

**Comment by koekeishiya** [comment]
I don't think there is an efficient way for yabai to be able to tell other windows that they should be blurred. There are private APIs to do this: CGSSetWindowBackgroundBlurRadius but I don't know how taxing that is performance wise.

**Comment by koekeishiya** [comment]
It is not possible to properly blur arbitrary windows. While we do have authorization to call the function mentioned above through the scripting-addition, it is not a viable solution for the following reason:

The only way can adjust the alpha of some window is through a call to the WindowServer and I assume that this works by modifying properties of the final composite image before it is drawn to the screen, and so at the time we try to apply a blur effect, the window is not actually transparent and the blur can't/won't be applied.

I verified my above assumptions by creating a non-opaque window and trying to fill the context with a 50% transparent color and noticed that it did not work (as expected). The call to the WindowServer however, is still able to make the window draw transparently on the screen. Applying blur using the above API does not work here.

I then made an opaque window, did not use the WindowServer to make it transparent, but set transparency directly in the color value used to draw the background. Window appears transparent (as expected). Applying blur using the above API call worked fine here.
